pom.xml

```diff
@@ -50,6 +50,11 @@
 		    <artifactId>lombok</artifactId>
 		    <version>1.18.24</version>
 	    </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>1.4.11</version>
+        </dependency>
```

Comment:
By giving Slf4j a logger to use (logback), this removed the no-op logger error message and also gave us console logs; useful for debugging.